Makefile.core.mk

```diff
@@ -35,10 +35,11 @@ gen-check: gen check-clean-repo
 
 containers:
 	@gcloud auth configure-docker -q	# enable docker to authenticate with gcr.io, needed for prow
-	@cd docker/build-tools && ./build-and-push.sh
+	@cd docker/build-tools && DRY_RUN=true CONTAINER_BUILDER="buildx build --push --platform=linux/amd64,linux/arm64" ./build-and-push.sh
```

Comment:
multi arch images release have to use `docker buildx build --push`.
can't store in local and `docker push` later.